src/Service.Tests/dab-config.PostgreSql.json

```diff
@@ -14,7 +14,12 @@
     "graphql": {
       "enabled": true,
       "path": "/graphql",
-      "allow-introspection": true
+      "allow-introspection": true,
+      "nested-mutations": {
```

Comment:
Doesnt the old config work as is? Why do we need to explicitly set this to false?

Reply:
The old config (without the `nested-mutations` section ) works correctly. Added the section in the reference config files to showcase how the section would look like